workers/unity/Packages/com.improbable.gdk.core/Utility/ViewCommandBuffer.cs

```diff
@@ -46,23 +46,23 @@ public void RemoveComponent(Entity entity, ComponentType componentType)
             });
         }
 
-        public void FlushBuffer(MutableView view)
+        public void FlushBuffer(WorkerBase worker)
```

Comment:
Pass in an `EntityManager` instead? 

The consumer of this buffer will (should) be inside a `ComponentSystem` so will have access to that directly

Reply:
then we can't use the logdispatcher anymore, unless we want to pass that one in directly as well

Reply:
Oh, that's probably fine then